notebooks/MAST/plot_exoplanet_lightcurves/plot_exoplanet_lightcurves.ipynb

```diff
@@ -0,0 +1,1678 @@
+{
```

Comment:
<p>This should maybe be a "helper function"; we want users to understand what happens, and we should explain what goes on here, but converting a JSON to astropy table is a "sweep it under the rug" step of the process. Basically just do something like:</p><pre class="ql-syntax" spellcheck="false">def JSON2aptable():
  function goes here
</pre><p>And explain to users that we're just converting between data types</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Great point! @kelle on previous notebooks you'd indicated that I might not want to introduce functions, because users might not be familiar with them. What are your thoughts here?</p>